packages/scrollbar/src/index.vue

```diff
@@ -28,7 +28,7 @@
 <script lang="ts">
 import { addResizeListener, removeResizeListener } from '@element-plus/utils/resize-event'
 import { addUnit, isArray, isString, toObject } from '@element-plus/utils/util'
-import { computed, defineComponent, nextTick, onBeforeUnmount, onMounted, provide, ref } from 'vue'
+import { computed, CSSProperties, defineComponent, nextTick, onBeforeUnmount, onMounted, PropType, provide, ref } from 'vue'
 import Bar from './bar.vue'
 
```

Comment:
Here: add it after line#33.

Reply:
OK，i will changed then created a new pr

Reply:
I have a question, I want to delete other commits, what do you suggest?